ds701_book/04-Linear-Algebra-Refresher.qmd

```diff
@@ -255,6 +257,19 @@ plt.show()
 
 Observe that a right angle forms between the vectors $\mathbf{u}$ and $\mathbf{v}$ when $\mathbf{u}\cdot \mathbf{v} = 0$. 
 
+```{python}
```

Comment:
Is this left over check of the previous code block for the dot product figure?

Reply:
yes, kind of. I want to figure out why plot works with the wrong norm equation. I can hide the cell or figure out the issue in the mean time.